tests/helpers/images.ts

```diff
@@ -0,0 +1,21 @@
+import { expect } from "../fixtures/lxd-test";
+import { Page } from "@playwright/test";
+
+export const deleteAllImages = async (
+  page: Page,
+  project: string = "default",
+) => {
+  await page.goto(`/ui/project/${project}`);
+  await page.getByRole("link", { name: "Images", exact: true }).first().click();
+  await page
+    .locator("#image-table .select-header input[type=checkbox]")
```

Comment:
aah yes, I was using the playwright pick selector functionality and couldn't find the input element that needs to be clicked. Turns out the input is position above the box and it's hidden. Double checked this code `await page.getByLabel("Select all").click();` works, and I think it may resolve the failing tests in firefox.

Reply:
Yeah, the playwright code generator does not always work that well.

Reply:
Okay a bit of an adjustment to this one, since now the styling is fixed, the label for the select checkbox actually completely overlays the checkbox. `getByLabel` will target the input, which can not be clicked, so we need to click the label which requires the below locator. Still using a11y properties though.

```
  await page
    .getByRole("columnheader", { name: "select" })
    .locator("label:has-text('Select all')")
    .click();
```